andela/neon-ah-backend
# #163620272 Documented all endpoints on Swagger

#### What does this PR do?
 - This PR documents all the endpoints of our REST API

#### How should this be manually tested?
 - clone repo 
 - switch to this branch 
 - Launch on local
 - view docs on localhost URL/docs
 -  When merged Docs should be up live on https://neon-ah-staging.herokuapp.com/api/v1/docs/#/

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2229851/stories/163620272
